arangod/Cluster/Maintenance.cpp

```diff
@@ -1233,20 +1233,20 @@ static auto reportCurrentReplicatedLogLocal(replication2::replicated_log::LogSta
   return std::nullopt;
 }
 
-static auto reportCurrentReplicatedLogLeader(replication2::replicated_log::LeaderStatus const& status,
+static auto reportCurrentReplicatedLogLeader(replication2::replicated_log::QuickLogStatus const& status,
                                              replication2::agency::LogCurrent::Leader const* currentLeader)
     -> std::optional<replication2::agency::LogCurrent::Leader> {
   if (status.leadershipEstablished) {
```

Comment:
We could assert here, that `status.role == kLeader`?

Reply:
That is a good idea!